src/styles/RulesRankingStart.scss

```diff
@@ -0,0 +1,24 @@
+@use './colors';
+@use './common';
+@use './typography';
+
+
+.mainpage-wrapper-rules-ranking-start {
```

Comment:
Uwaga kosmetyczna. Starajmy się pisać zwięźlej nazwy klas. Nie jest to łatwe w przypadku wielu zagnieżdżeń. Niektóre narzędzia pomagają/rozwiązują ten problem. Warto jednak zapoznać się z podstawami BEM.

Zapraszam WSZYSTKICH do lektury :D
https://www.nafrontendzie.pl/metodyki-css-2-bem